packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js

```diff
@@ -330,6 +357,14 @@ describe('plugin-meetings', () => {
         // Check that the mqe data has been emitted and is correctly computed.
         checkMqeData();
       });
+
+      it('emits the correct transportType in MEDIA_QUALITY events', async () => {
```

Comment:
this checks the protocol, that's great, but it would be nice to also have a test that checks the relayProtocol